app/src/script.js

```diff
@@ -1,18 +1,188 @@
-import Aragon from '@aragon/client'
+import 'core-js/stable'
+import 'regenerator-runtime/runtime'
+import { map, publishReplay } from 'rxjs/operators'
+import { of } from 'rxjs'
+import AragonApi from '@aragon/api'
 
-const app = new Aragon()
+import { addressesEqual } from './lib/web3-utils'
+import tokenAbi from './abi/token.json'
 
-const initialState = {
-  dummyValue: 0
+const INITIALIZATION_TRIGGER = Symbol('INITIALIZATION_TRIGGER')
```

Comment:
I should have done it when I uploaded the base app but if you upgrade to the latest api you can specify an init state directly in api.store(), amongst other small changes which I can't remember right now. 

eg:

    "@aragon/api": "^2.0.0-beta.4",
    "@aragon/api-react": "^2.0.0-beta.4"